web

```diff
@@ -0,0 +1 @@
+webui/react
```

Comment:
should we just move `webui/react` over to `web`?

Reply:
💯

Reply:
yeah i would really like for that to happen as well.

only thing is just to merge `webui/Makefile` and `webui/react/Makefile`